src/Goteo/Model/Faq/FaqSection.php

```diff
@@ -111,63 +98,82 @@ static public function getList(array $filters = [], int $offset = 0, int $limit
               $sql
               ORDER BY faq_section.order ASC
               LIMIT $offset, $limit";
-         //die(\sqldbg($sql, $values));
         $query = static::query($sql, $values);
         return $query->fetchAll(\PDO::FETCH_CLASS, __CLASS__);
     }
 
+     static public function getListCount(array $filters = [], $lang = null): int
+     {
+         if(!$lang) $lang = Lang::current();
+         list($fields, $joins) = self::getLangsSQLJoins($lang, Config::get('sql_lang'));
+
+         $filter = [];
+         $values = [];
+
+         if($filter) {
+             $sql = " WHERE " . implode(' AND ', $filter);
+         }
+
+         $sql = "SELECT
+              count(faq_section.id)
+          FROM faq_section
+          $joins
+          $sql";
+         $query = static::query($sql, $values);
+         return $query->fetchColumn();
+    }
 
-  public function getFaqbySection(int $limit = 0): array
-  {
-
-    return Faq::getList(['section'=>$this->id], 0, $limit);
-
-  }
+    /**
+     * @return Faq[]
+     */
+    public function getFaqs(int $limit = 0): array
+    {
+        $count = Faq::getListCount(['section' => $this->id]);
+        return Faq::getList(['section'=>$this->id], 0, $count);
+    }
 
 
- public function getBannerHeaderImage(): Image {
+    public function getBannerHeaderImage(): Image {
       if(!$this->bannerHeaderImageInstance instanceOf Image) {
           $this->bannerHeaderImageInstance = new Image($this->banner_header_image);
       }
 
       return $this->bannerHeaderImageInstance;
-  }
-
-  public function save(&$errors = array()): bool
-  {
-
-    if (!$this->validate($errors))
-        return false;
-
-    $fields = [
-        'id',
-        'name',
-        'slug',
-        'icon',
-        'banner_header',
-        'button_action',
-        'button_url',
-        'lang',
-        'order'
-        ];
-
-    try {
-        //automatic $this->id assignation
-        $this->dbInsertUpdate($fields);
-
-        return true;
-    } catch(\PDOException $e) {
-        $errors[] = "Faq section save error: " . $e->getMessage();
-        return false;
     }
-  }
 
-    public function validate(&$errors = array()): bool {
+    public function save(&$errors = array()): bool
+    {
+        if (!$this->validate($errors))
+            return false;
+
+        $fields = [
+            'id',
+            'name',
+            'slug',
+            'icon',
+            'banner_header',
+            'button_action',
+            'button_url',
+            'lang',
+            'order'
+            ];
```

Comment:
@davidbeig I propose you to move the `$fields` variable of this method, into a class constant. With a name that highlights the purpose of properties being saved.

This way, the method is much smaller, and perhaps the constant can be reused somewhere else.

Something like:

```
const EDITABLE_PROPERTIES = [
            'id',
            'name',
            'slug',
            'icon',
            'banner_header',
            'button_action',
            'button_url',
            'lang',
            'order'
        ];
```